prestoadmin/package.py

```diff
@@ -92,12 +92,20 @@ def rpm_install(rpm_name):
     nodeps = ''
     if env.nodeps:
         nodeps = '--nodeps '
-
-    ret = sudo('rpm -i %s%s' %
-               (nodeps, os.path.join(constants.REMOTE_PACKAGES_PATH,
+   
+    if env.java8_home is None:
+        ret = sudo('rpm -i %s%s' %
+                   (nodeps, os.path.join(constants.REMOTE_PACKAGES_PATH,
+                                     rpm_name)))
+        if ret.succeeded:
+            print("Package installed successfully on: " + env.host)
```

Comment:
Nit: The check on ret should be factored out of the if block so it isn't duplicated.